### PR TITLE
Graph Management

### PR DESCRIPTION
**Pull request by EpiFouloux**
This Pull request provides basic named graph management:
- Creation
- Deletion
- Retrieval by id or global retrieval

Based on https://www.arangodb.com/docs/stable/http/gharial-management.html

Missing: 
- Graph edition (adding/removing vertex and edge collections)
- Edge collection retrieval
- Vertex collection retrieval

### CODE REVIEW

**Comment by EpiFouloux** [comment]
Hey @inzanez @fMeow, @Weasy666 wanted named graph handling so I made this basic implementation inspired by the index management PR.

For some reason my tests don't launch in local and github is saying my build is failing but I don't get why

File: src/graph.rs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+//!
+//! The various structures are following the HTTP specification as detailed in this ArangoDB
+//! [section](https://www.arangodb.com/docs/stable/http/gharial-management.html)
+//!
+//! For detailed information about ArangoDB named graphs, please check out the official
+//! ArangoDB [documentation](https://www.arangodb.com/docs/stable/http/gharial.html).
+use serde::{Deserialize, Serialize};
+use typed_builder::TypedBuilder;
+
+pub const GHARIAL_API_PATH: &str = "_api/gharial";
```
Comment (fMeow) [inline src/graph.rs:12]: Is `GHARIAL_API_PATH` a typo? Or am I missed something?

Comment (EpiFouloux) [reply src/graph.rs]: For some reason it's called [gharial](https://www.arangodb.com/docs/stable/http/gharial-management.html) but I can rename it to "graph" for the `const`.

I changed both new `const` into `pub(crate)` as you requested

Comment (fMeow) [reply src/graph.rs]: That is fine, don't need to rename. Thank you.

**Pull request merged**